### DISCUSSION TITLE
计算引擎有考虑使用WebAssembly来提升性能吗？

### DISCUSSION THREAD

**Discussion by AquaHydro** [dream-num/univer#2932] [Ideas | 产品建议]
如题，参考一下[https://juejin.cn/post/7207079381670314041](https://juejin.cn/post/7207079381670314041)

**Comment by jikkai** [comment]
If you have used [the plugins](https://github.com/dream-num/obsidian-univer) we provide for Obsidian, you should have noticed that we have started using wasm. So the answer is: **yes**, we are considering it, but specific plans need to go through a series of tests and verifications before they can be confirmed. There will not be clear planning in the short term :octocat: 

---

如果你有使用过我们为 Obsidian 提供的[插件](https://github.com/dream-num/obsidian-univer)的话，应该会注意到我们已经开始使用了 wasm。因此答案是：**是的**，我们有在考虑，但是具体的计划还要经过一系列的测试和验证才能确定，短期内不会有明确的规划。 :octocat:
